.github/workflows/release.yml

```diff
@@ -12,13 +12,19 @@ on:
         description: 'HomeBrew prerelease'
         default: 'false'
 jobs:
-  token-validation:
+  cli-core-token-validation:
     runs-on: ubuntu-latest
     steps:
+      - name: Trigger CLI token validation workflow
+        uses: benc-uk/workflow-dispatch@v1
```

Comment:
Any reason for not using [trigger-workflow](https://github.com/twilio/twilio-cli-core/blob/main/.github/scripts/trigger-workflow.js) instead of `benc-uk/workflow-dispatch@v1`

Reply:
This just seemed more simple and straightforward

Reply:
the reason we implemented [trigger-workflow](https://github.com/twilio/twilio-cli-core/blob/main/.github/scripts/trigger-workflow.js) script was to move away from community actions. Using it should be straightforward as well. One example is in the [PR](https://github.com/twilio/twilio-cli/pull/392/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R209-R215) itself that @shrutiburman raised

Reply:
Changed this to use trigger-workflow

Reply:
New test run: https://github.com/shamantraghav/twilio-cli-core/actions/runs/1828065685